doc/make.py

```diff
@@ -326,7 +214,7 @@ def main():
                            help='command to run: {}'.format(', '.join(cmds)))
     argparser.add_argument('--num-jobs',
                            type=int,
-                           default=1,
+                           default=0,
```

Comment:
In the DocBuilder init, the default is 1. Make them both the same?

What does a value of 0 do?

Reply:
Good point. with `0` I don't pass a `-j` value to `sphinx-build`, while with `1`, I pass `-j 1`, which is actually the same (unless sphinx changes its default).

It's a bit weird, because the `-j` parameter is supposed to make `sphinx-build` run with multiple cores, but it actually doesn't work, and the process takes exactly the same with `-j 1` and `-j 4`. It could make sense to remove this option, but I guess at some point sphinx will be fixed at some point, so I guess it's worth leaving it.

I'm happy setting our default to `0` (use sphinx default), `1`, or `auto`, just let me know if you have a preference.

I address your other comment.

Reply:
Hmm I think multiple cores speeds things up for me, at least for a full doc build.

Reply:
I checked it with `time` in Linux, 1 vs 4 cores, it took 17m30s in both cases (with a difference of less than 5 seconds between them). I tried it in the past with a different computer (also Linux) and was the same.